src/pl_bolts/models/self_supervised/swav/swav_module.py

```diff
@@ -158,6 +159,16 @@ def init_model(self):
             backbone = resnet18
         elif self.arch == "resnet50":
             backbone = resnet50
+        elif self.arch == "swin_s":
```

Comment:
lets change this to mapping

Reply:
I changed it into a dictionary instead of multiple if conditions